lib/redtastic/scripts/hmsum.lua

```diff
@@ -0,0 +1,31 @@
+-- Returns the total sum of the values for each key in KEYS, across multiple ids in ARGV
+-- KEYS: [ "key1", "key2" ]
+-- ARGV: [1, 2, 3]
+
+-- Example data structures
+-- hkeys key1
+-- => "1" => "1", "2" => "4"
+-- hkeys key2
+-- => "2" => "3", "3" => "5"
+-- hmget key1 1 2 3
+-- => ["1", "4", nil]
+-- hmget key2 1 2 3
+-- => [nil, "3", "5"]
+
+-- This script will evaluate the above data structure and sum the result of the array values
+-- that match the ids in ARGV. i.e. The example above will return 5 + 8 = 13
+
+local sum = 0
+
+for i=1,#KEYS do
```

Comment:
@sisk, according to [this](https://springrts.com/wiki/Lua_Performance#TEST_9:_for-loops), that's the most efficient way to loop through an array, faster than using `ipairs()`, which was overkill for our use case.